ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
+        """ negative test """
+        self.assertNotEqual(analyzer.word_count, {'test': 1})
+
+    def test_analyze_comments(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        result = analyzer.analyzeComment("This is a test comment")
+        self.assertEqual(result, {'test': 1, 'Sentiment': 0.0, 'Code Blocks': 0})
+        """ negative test """
+        self.assertNotEqual(result, {'test': 0, 'Sentiment': 0.0, 'Code Blocks': 0})
+
+    def test_preProcess(self):
+        analyzer = CommentAnalyzer(['test'])
+        result = analyzer.preProcess("This is a test comment")
+        """ positive test """
+        self.assertEqual(result, "this is a test comment")
+        """ negative test """
+        self.assertNotEqual(result, "This is a test comment")
+    
+    def test_countWords(self):
+        analyzer = CommentAnalyzer(['test'])
+        result = analyzer.countWords("This is a test comment")
+        """ positive test """
+        self.assertEqual(result, {'test': 1})
+        """ negative test """
+        self.assertNotEqual(result, {'test': 0})
+    
+    def test_getSentiment(self):
+        analyzer = CommentAnalyzer(['test'])
+        result = analyzer.getSentiment("This is a test comment")
+        """ positive test """
+        self.assertEqual(result, 0.0)
+        """ negative test """
+        self.assertNotEqual(result, 0.1)
```

Comment:
Since the sentiment value could change in the future based on the behavior of Vader, I wouldn't hard code the scores here. Would it make more sense to mock the call to return a specific value to test the functionality that it calls `polarity_scores`?